packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/PublishAction.tsx

```diff
@@ -248,18 +248,10 @@ const SelectedEntriesTableContent = ({
                     search: row.locale && `?plugins[i18n][locale]=${row.locale}`,
                   }}
                   state={{ from: pathname }}
-                  label={formatMessage(
-                    { id: 'app.component.HelperPluginTable.edit', defaultMessage: 'Edit {target}' },
-                    {
-                      target: formatMessage(
-                        {
-                          id: 'content-manager.components.ListViewHelperPluginTable.row-line',
-                          defaultMessage: 'item line {number}',
-                        },
-                        { number: index + 1 }
-                      ),
-                    }
-                  )}
+                  label={formatMessage({
```

Comment:
I suggest you to use something like
`label={formatMessage({
    id: 'content-manager.bulk-publish.edit',
    defaultMessage: 'Edit item',
 })}`
 
 and add the new id here https://github.com/strapi/strapi/blob/develop/packages/core/admin/admin/src/translations/en.json